File: python/darts/darts.py
from math import sqrt
def score(x, y):
    inner, middle, outer = 1, 5, 10
    dart = sqrt(x**2+ y**2)
    if dart <= inner:
        return 10
    if dart <= middle:
        return 5
    if dart <= outer:
        return 1
    return 0

File: java/darts/src/main/java/Darts.java
class Darts {

    private final double x;
	private final double y;
	
    Darts(double x, double y) {
    	this.x = x;
    	this.y = y;
    }

    int score() {
        double d = Math.sqrt( x * x + y * y );
        
        if (d <= 10 && d > 5) {
        	return 1;
        } else if (d <= 5 && d > 1) {
        	return 5;
        } else if (d <= 1) {
        	return 10;
        }
        
        return 0;
    }

}
